### PR TITLE
Hide delete button on own subuser if for the subuser looking at the page

### PR DESCRIPTION
**Pull request by notAreYouScared**
Don't show the delete button if a sub-user is looking at sub-users, as they cannot delete themselves. goes along with them not being able to edit themselves that's already a thing.

### CODE REVIEW

**Comment by DaneEveritt** [comment]
@AreYouRlyScared can you put the `if` check outside the `<Can/>` component, just logically makes more sense to me and follows the design for other component stuff. 👍

**Comment by notAreYouScared** [comment]
@DaneEveritt Done.

File: resources/scripts/components/server/users/UserRow.tsx [hunk 1]
```diff
@@ -48,23 +48,23 @@ export default ({ subuser }: Props) => {
...
-                    <FontAwesomeIcon icon={faPencilAlt}/>
-                </button>
-                }
-            </Can>
-            <Can action={'user.delete'}>
-                {subuser.uuid !== uuid &&
-                <RemoveSubuserButton subuser={subuser}/>
-                }
-            </Can>
+            {subuser.uuid !== uuid &&
```
Comment (DaneEveritt) [inline resources/scripts/components/server/users/UserRow.tsx:51]:
```
{subuser.uuid !== uuid &&
    <>
        {the two <can> blocks}
    </>
}
```

**Review by DaneEveritt — CHANGES_REQUESTED** [review]

**Review by notAreYouScared — COMMENTED** [review]
Changed

**Pull request merged**